tests/libtest/CMakeLists.txt

```
@@ -30,7 +30,7 @@ set_source_files_properties("../../lib/curl_multibyte.c" PROPERTIES SKIP_UNITY_B
 
 add_custom_command(
   OUTPUT "lib1521.c"
-  COMMAND ${PERL_EXECUTABLE} "${CMAKE_CURRENT_SOURCE_DIR}/mk-lib1521.pl" < "${CURL_SOURCE_DIR}/include/curl/curl.h" > "lib1521.c"
+  COMMAND ${PERL_EXECUTABLE} "${CMAKE_CURRENT_SOURCE_DIR}/mk-lib1521.pl" < "${CURL_SOURCE_DIR}/include/curl/curl.h" > "lib1521.tmp" && mv lib1521.tmp lib1521.c
```

```suggestion
  COMMAND ${PERL_EXECUTABLE} "${CMAKE_CURRENT_SOURCE_DIR}/mk-lib1521.pl" < "${CURL_SOURCE_DIR}/include/curl/curl.h" > "lib1521.tmp"
  COMMAND ${CMAKE_COMMAND} -E rename "lib1521.tmp" "lib1521.c"
```
This makes it portable with any shells.

https://cmake.org/cmake/help/latest/manual/cmake.1.html#cmdoption-cmake-E-arg-rename

---

I moved the rename into the perl script instead

---

Even better!